docs/creating-editing-and-removing-specialist-document-types-and-finders.md

```diff
@@ -9,15 +9,11 @@ applications.
 
 # __Creating__ a specialist document type
 
-<!-- TODO: Update example links in the documentation, to reflect latest documentation. Some of the steps are now no longer needed, but example PRs are outdated since no new requests for changes have been made. Also, remove the notes about not adding enums, once all examples are updated. -->
-
 ## 1. Add a schema to Publishing API
-See [example PR here](https://github.com/alphagov/publishing-api/pull/2589/files)
```

Comment:
I think https://github.com/alphagov/publishing-api/pull/3026 would be a better choice here, as it shows custom metadata fields that PR 2943 does not.